1st Year/05. Binary Search in 1D Array/Binary_search_in_1D_array.py

```diff
@@ -0,0 +1,57 @@
+## To implement binary search technique in 1D Array
+from bisect import bisect_left
```

Comment:
Do not use in-built function ```bisect_left()```. You are expected to implement the traditional binary search algorithm.

Reply:
solved